site/profile/manifests/core/anaconda.pp

```diff
@@ -0,0 +1,78 @@
+# @summary
+#   Installs anaconda
```

Comment:
I still feel that this would be better as an external module instead of as a profile as the functionality isn't "site" specific.

Reply:
I'm investigating how to do it, but I still want to get the other details reviewed, please.